docs/docs/classic-ml/tracking/backend-stores/index.mdx

```diff
@@ -0,0 +1,136 @@
+import Link from "@docusaurus/Link";
+import { APILink } from "@site/src/components/APILink";
+
+# Backend Stores
+
+The backend store is a core component in [MLflow Tracking](/ml/tracking) where MLflow stores metadata for
+[Runs](/ml/tracking#runs) and experiments such as:
+
+- Run ID
+- Start & end time
+- Parameters
+- Metrics
+- Code version (only if you launch runs from an [MLflow Project](/ml/projects)).
+- Source file name (only if you launch runs from an [MLflow Project](/ml/projects)).
+
+Note that large model artifacts such as model weight files are stored in [artifact store](/ml/tracking/artifacts-stores).
+
+## Configure Backend Store
+
+By default, MLflow stores metadata in local files in the `./mlruns` directory, but MLflow can store metadata to databases as well.
+You can configure the location by passing the desired **tracking URI** to MLflow, via either of the following methods:
+
+- Set the `MLFLOW_TRACKING_URI` environment variable.
+- Call <APILink fn="mlflow.set_tracking_uri" /> in your code.
+- If you are running a [Tracking Server](/ml/tracking#tracking_server), you can set the `tracking_uri` option when starting the server, like `mlflow server --backend-store-uri sqlite:///mydb.sqlite`
+
+Continue to the next section for the supported format of tracking URLs.
+Also visit [this guidance](/ml/tracking#tracking_setup) for how to set up the backend store properly for your workflow.
+
+## Supported Store Types
+
+MLflow supports the following types of tracking URI for backend stores:
+
+- Local file path (specified as `file:/my/local/dir`), where data is just directly stored locally to a system disk where your code is executing.
+- A Database, encoded as `<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>`. MLflow supports the dialects `mysql`, `mssql`, `sqlite`, and `postgresql`. For more details, see [SQLAlchemy database uri](https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls).
+- HTTP server (specified as `https://my-server:5000`), which is a server hosting an [MLflow tracking server](/ml/tracking#tracking_server).
+- Databricks workspace (specified as `databricks` or as `databricks://<profileName>`, a [Databricks CLI profile](https://github.com/databricks/databricks-cli#installation)).
+  Refer to Access the MLflow tracking server from outside Databricks [[AWS]](http://docs.databricks.com/applications/mlflow/access-hosted-tracking-server.html)
+  [[Azure]](http://docs.microsoft.com/azure/databricks/applications/mlflow/access-hosted-tracking-server), or [the quickstart](/ml/getting-started/intro-quickstart) to
+  easily get started with hosted MLflow on the Databricks Free Trial.
+
+:::warning important
+In order to use [Model Registry](/ml/model-registry) functionality, you must run your server using a database-backed store. See [this FAQ](/ml/tracking#tracking-with-model-registry) for more information.
+:::
+
+:::warning important
```

Comment:
This looks a bit weird to have two warnings back to back, maybe merge them into one?